bindings/responses.go

```diff
@@ -11,8 +11,9 @@ import (
 
 // ReadResponse is the return object from an dapr input binding.
 type ReadResponse struct {
-	Data     []byte            `json:"data"`
-	Metadata map[string]string `json:"metadata"`
+	Data        []byte            `json:"data"`
+	Metadata    map[string]string `json:"metadata"`
+	ContentType *string           `json:"contentType,omitempty"`
```

Comment:
Nit: You can use `string` instead of `*string` here. The zero value of a string in Go works well here. Same goes for all the other occurrences below.

Reply:
This is something I asked for to keep it similar to how eTag is done.

Reply:
resolving the comment as per above comment